config.sh.SAMPLE

```diff
@@ -42,6 +42,8 @@ gitlab_url="https://gitlab.example.com"
 gitlab_user="gitmirror"
 #Generate a token for your $gitlab_user and set it here.
 gitlab_user_token_secret="$(head -n1 "${user_home}/private_token" 2> /dev/null || echo "")"
+#Sets the Gitlab API version, either 3 or 4
+gitlab_api_version=3
```

Comment:
Default to 4. Version 4 was supported in Gitlab 9.

Reply:
I am OK with defaulting to v4.